ocaml/typing/jkind.mli

```diff
@@ -406,6 +526,49 @@ val format_history :
     this module. *)
 val set_printtyp_path : (Format.formatter -> Path.t -> unit) -> unit
 
+(******************************)
+(* errors *)
+
+module Violation : sig
```

Comment:
I don't think we want both a `Type.Violation` and a `Violation`. Just keep this outer one.